docs/InstallGuide.md

```diff
@@ -39,17 +34,10 @@ Install epel-release.
     # rpm -Uvh http://dlc.wakame.axsh.jp.s3-website-us-east-1.amazonaws.com/epel-release
 
 
-OpenVNet installation
-------------------------
-
-Install OpenVNet.
+Install OpenVNet packages.
 
     # yum install -y openvnet
```

Comment:
Before installing this package I would include a brief explanation that OpenVNet consists of 3 parts (vnmgr, webapi, vna) and that this is a meta-package that depends on them all.

Kind of like how the Wakame install guide explains the different parts http://axsh.github.io/wakame-vdc/installation/